README.md

```diff
@@ -259,6 +261,22 @@ String transactionHash = xrpClient.send(amount, destinationAddress, wallet);
 
 **Note:** The above example will yield an "Account not found." error because the randomly generated wallet contains no XRP.
 
+### Enabling Deposit Authorization
+
+```java
+
+String grpcURL = "test.xrp.xpring.io:50051"; // Testnet URL, use main.xrp.xpring.io:50051 for Mainnet
+XrpClient xrpClient = new XrpClient(grpcURL, XrplNetwork.TEST);
+
+// Wallet for which to enable Deposit Authorization
+Wallet seedWallet = new Wallet("snRiAJGeKCkPVddbjB3zRwwoiYDBm1M");
+
+TransactionResult transactionResult = xrpClient.enableDepositAuth(seedWallet);
+String transactionHash = transactionResult.hash;
```

Comment:
I think you need to call this `hash()` instead of `hash` since it's an Immutable right?